File: 1323. Maximum 69 Number.cpp
class Solution {
public:
    int maximum69Number (int num) {
      
        string str = to_string(num);// convert to string

        for(int i=0; i<str.size(); i++){
            if(str[i] == '6'){
                str[i] = '9';
                break;
            }
        }
        return stoi(str); //string to int
    }
};

File: 1323. Maximum 69 Number.java
class Solution {
    public int maximum69Number (int num) {
        String str = Integer.toString(num);
        char[] arr = str.toCharArray();

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == '6') {   // change first 6 to 9
                arr[i] = '9';
                break;
            }
        }

        return Integer.parseInt(new String(arr));
    }
}
